src/Flatfile.ts

```diff
@@ -140,8 +141,16 @@ export class Flatfile extends TypedEventManager<IEvents> {
         this.emit('launch', { batchId: meta?.batchId })
       }, 0)
 
-      if (options?.open === 'iframe') {
-        const importFrame = session.openInEmbeddedIframe({ autoContinue: options?.autoContinue })
+      if (mountOn || options?.open === 'iframe') {
+        if (mountOn && !existsInDOM(mountOn, 'iframe')) {
```

Comment:
should `mountOn` be a selector for the container to which we should append our iframe (instead of being a selector to an iframe element)?

Reply:
@juribee i would probably vote for a container to which we're gonna append our iframe. This way we don't have to worry about customer possibly updating the iframe after our mount